src/utils.rs

```diff
@@ -243,14 +244,31 @@ pub fn wrap_seq_for_shell(
 
 fn internal_exec_cmd(cmd: &str, args: &[&str]) -> Option<CommandOutput> {
     log::trace!("Executing command {:?} with args {:?}", cmd, args);
-    match Command::new(cmd).args(args).output() {
+
+    let full_path = match which::which(cmd) {
```

Comment:
It might be worth it to build a map of the whole `PATH` in the future, to avoid having to walk the whole `PATH` on each invocation.

Reply:
I think that might be something to look at in the future or maybe something we can get added to the upstream library.